src/components/organisms/ProfileModal/UserInformationContent/UserInformationContent.tsx

```diff
@@ -150,9 +150,9 @@ export const UserInformationContent: React.FunctionComponent<UserInformationCont
           >
             {email}
           </div>
-          {sovereignVenue?.showUserStatus && (
+          {sovereignVenue?.showUserStatus && sovereignVenue?.userStatuses && (
```

Comment:
looks like `<UserStatusDropdown />` breaks when `userStatuses` list is empty (length === 0).
maybe you were right adding this check to the `< UserStatusDropdown />` component;

so we need to check both cases: `userStatuses is undefined`, `userStatuses is []`

Reply:
@denisdimitrov and @sunny-viktoryia what's up here? Can we finalize plz?

Reply:
> looks like <UserStatusDropdown /> breaks when userStatuses list is empty (length === 0).
maybe you were right adding this check to the < UserStatusDropdown /> component;

so we need to check both cases: userStatuses is undefined, userStatuses is []

It's probably because I'm checking if `sovereignVenue?.userStatuses` exists. Which will pass as `true` even if the array is empty. I think this should work If I check for the length instead.

Reply:
@sunny-viktoryia Fixed in: https://github.com/sparkletown/sparkle/pull/1772/commits/832b2e1e6e77cfefb88accaa67edcb3cc3389988